src/extension/wps/wps-core/src/main/java/org/geoserver/wps/validator/MultiplicityValidator.java

```diff
@@ -0,0 +1,88 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wps.validator;
+
+import org.hsqldb.lib.Collection;
+import org.springframework.validation.Errors;
+
+/**
+ * Check there are not too many instances of the input
+ * 
+ * @author Andrea Aime - GeoSolutions
+ */
+public class MultiplicityValidator implements WPSInputValidator {
```

Comment:
Does this need to be public? Or can client code use ObjectSizeEstimators utility methods?